sktm/db.py

```diff
@@ -455,23 +456,21 @@ def dump_baseline_tests(self):
                          'baseline.testrun_id = testrun.id')
 
         for (burl, commit, res, buildid) in self.cur.fetchall():
-            print "repo url: %s" % burl
-            print "commit id: %s" % commit
-            print "result: %s" % sktm.tresult(res).name
-            print "build id: #%s" % buildid
-            print "---"
+            print("repo url:", burl)
+            print("commit id:", commit)
+            print("result:", sktm.tresult(res).name)
+            print("build id: #", buildid, sep='')
+            print("---")
 
     def dump_baserepo_info(self):
         self.cur.execute('SELECT url FROM baserepo')
 
         for (burl,) in self.cur.fetchall():
-            print "repo url: %s" % burl
+            print("repo url:", burl)
             stable = self.get_stable(burl)
             latest = self.get_latest(burl)
-            print "most recent stable commit: %s (%s)" % (
-                stable, self.get_commitdate(burl, stable)
-            )
-            print "most recent tested commit: %s (%s)" % (
-                latest, self.get_commitdate(burl, latest)
-            )
-            print "---"
+            print("most recent stable commit: ", stable, " (",
+                  self.get_commitdate(burl, stable), ")", sep='')
+            print("most recent tested commit: ", latest, " (",
+                  self.get_commitdate(burl, latest), ")", sep='')
```

Comment:
Since these two statements are a bit more complicated, I believe it would be more readable to use `.format` to format the string with needed parameters and print directly that, instead of putting it together in the `print` function itself, what do you say?